common/components/org.wso2.carbon.data.agents.log4j/src/main/java/org/wso2/carbon/data/agents/log4j/appender/LogEventAppender.java

```diff
@@ -310,11 +310,11 @@ private void publishLogEvent(TenantAwareLoggingEvent event) throws ParseExceptio
                                 .getMessage(), e);
             } catch (DataEndpointAuthenticationException e) {
                 log.error(
-                        "Invalid urls passed for receiver and auth, and hence expected to fail " + e
+                        "Error while trying to login to data receiver : " + e
                                 .getMessage(), e);
             } catch (TransportException e) {
                 log.error(
-                        "Invalid urls passed for receiver and auth, and hence expected to fail " + e
+                        "Thrift transport exception occurred " + e
                                 .getMessage(), e);
```

Comment:
Hi Niranjan,

This is fixed in the commit : https://github.com/wso2/shared-analytics/pull/113/files#diff-d45f010b040aded5b5711f7ddab28e59R176

there use logError method to handle the log appender logs in common place,

https://github.com/wso2/shared-analytics/pull/113/files#diff-d45f010b040aded5b5711f7ddab28e59R260

Thanks,
SajithD